packages/flutter/lib/src/widgets/container.dart

```diff
@@ -427,7 +431,7 @@ class Container extends StatelessWidget {
       current = Padding(padding: margin!, child: current);
 
     if (transform != null)
-      current = Transform(transform: transform!, child: current);
+      current = transformAlignment == null ? Transform(transform: transform!, child: current) : Transform(transform: transform!, child: current, alignment: transformAlignment!);
```

Comment:
Can this just be:

```suggestion
      current = Transform(transform: transform!, child: current, alignment: transformAlignment);
```

?